zerver/lib/import_realm.py

```diff
@@ -848,14 +854,26 @@ def process_emojis(
             # This will overwrite the files that got copied to the appropriate paths
             # for emojis (whether in S3 or in the local uploads dir), ensuring to
             # thumbnail them and generate stills for animated emojis.
-            upload_emoji_image(f, record["file_name"], user_profile, content_type)
+            is_animated = upload_emoji_image(f, record["file_name"], user_profile, content_type)
         except BadImageError:
             logging.warning(
                 "Could not thumbnail emoji image %s; ignoring",
                 record["s3_path"],
             )
             # TODO:: should we delete the RealmEmoji object, or keep it with the files
             # that did get copied; even though they do generate this error?
+            return
+
+    if is_animated and not record.get("deactivated", False):
+        # We only update the is_animated field if the emoji is not deactivated.
+        # That's because among deactivated emojis (name, realm_id) may not be
+        # unique, making the implementation here a bit hairier.
+        # Anyway, for Zulip exports, is_animated should be set correctly from the start,
+        # while 3rd party exports don't use the deactivated field, so this shouldn't
+        # particularly matter.
+        RealmEmoji.objects.filter(
+            name=record["name"], realm_id=user_profile.realm_id, deactivated=False
+        ).update(is_animated=True)
```

Comment:
This is (3) from the commit message:
```
3. Finally, we notice that the 3rd party converters create all the
records with is_animated=False. That's an issue, because without setting
that correctly on the RealmEmoji objects, Zulip doesn't know that it
should use the "still" thumbnail when the emoji is being used in a
user's status. Which leads to incorrectly displaying the user status
with the distracting animation.
```